content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md

```diff
@@ -0,0 +1,452 @@
+
+The following VolumeSnapshotContent, for example exposes a volume snapshot with the name `7bdd0de3-aaeb-11e8-9aae-0242ac110002` belonging to a CSI driver called `testdriver.csi.k8s.io`.
+
+A VolumeSnapshotContent object should be created by a cluster admin with the following fields to represent an existing snapshot:
+
+- `driver` - CSI driver used to handle this volume. This field is required.
+- `source` - Snapshot identifying information
+- `snapshotHandle` - name/identifier of the snapshot. This field is required.
+- `volumeSnapshotRef` - Pointer to the VolumeSnapshot object this content should bind to.
+- `name` and `namespace` - Specifies the name and namespace of the VolumeSnapshot object which the content is bound to.
+- `deletionPolicy` - Valid values are `Delete` and `Retain`. If the `deletionPolicy` is `Delete`, then the underlying storage snapshot will be deleted along with the VolumeSnapshotContent object. If the - `deletionPolicy` is `Retain`, then both the underlying snapshot and VolumeSnapshotContent remain.
+
+```yaml
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotContent
+metadata:
+  name: manually-created-snapshot-content
+spec:
+  deletionPolicy: Delete
+  driver: testdriver.csi.k8s.io
+  source:
+    snapshotHandle: 7bdd0de3-aaeb-11e8-9aae-0242ac110002
+  volumeSnapshotRef:
+    name: test-snapshot
+    namespace: default
+```
+
+Once a VolumeSnapshotContent object is created, a user can create a VolumeSnapshot object pointing to the VolumeSnapshotContent object. The name and namespace of the VolumeSnapshot object must match the name/namespace specified in the volumeSnapshotRef of the VolumeSnapshotContent. It specifies the following fields:
+`volumeSnapshotContentName` - name of the volume snapshot content specified above. This field is required.
+`volumeSnapshotClassName` - name of the volume snapshot class. This field is optional.
+
+```yaml
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot
+metadata:
+  name: manually-created-snapshot
+spec:
+  source:
+        volumeSnapshotContentName: test-content
+```
+
+Once both objects are created, the common snapshot controller verifies the binding between VolumeSnapshot and VolumeSnapshotContent objects is correct and marks the VolumeSnapshot as ready (if the CSI driver supports the `ListSnapshots` call, the controller also validates that the referenced snapshot exists). The CSI external-snapshotter sidecar checks if the snapshot exists if ListSnapshots CSI method is implemented, otherwise it assumes the snapshot exists. The external-snapshotter sidecar sets `readyToUse` to true in the status field of VolumeSnapshotContent. The common snapshot controller marks the snapshot as ready accordingly.
+Create Volume From Snapshot
```

Comment:
"Create Volume From Snapshot" should be a heading.

Reply:
Change made and submitted a PR: https://github.com/rawkode/website/pull/1